pyschema/contrib/__init__.py

```diff
@@ -0,0 +1,8 @@
+import warnings
+
+warnings.warn(
+    "pyschema.contrib is deprecated and will be removed.\n"
+    "Please use the pyschema_extensions package instead.",
+    FutureWarning,
```

Comment:
Are you using `FutureWarning` just because `DeprecationWarning` won't bubble up?

Btw, hiding the warnings for deprecation is a strong candidate for the most stupid thing Python ever did..